addons/pos_restaurant_adyen/__manifest__.py

```diff
@@ -12,7 +12,7 @@
     'data': [
         'views/pos_payment_method_views.xml',
         ],
-    'auto-install': True,
+    'auto_install': True,
```

Comment:
ping @pimodoo @odoo/rd-payment might be interested in pos_restaurant_adyen having never auto-installed.